Dockerfile.prod

```diff
@@ -0,0 +1,16 @@
+FROM node:latest
+
+ENV NODE_ENV="production"
+
+WORKDIR /server
+
+RUN git clone https://github.com/codeuino/social-platform-donut-backend.git
+
+WORKDIR /server/social-platform-donut-backend
+
+RUN npm install && \
+    npm install pm2@latest -g && \
+    npm cache clean --force --loglevel=error
+
+# Start the server
+CMD [ "pm2", "start", "./bin/www", "--time", "--no-daemon" ]
```

Comment:
`pm2` process manager would give another wrapping for our node server to keep running all time (production use case)